### ISSUE TITLE
Atom 1.23.2 doesn't copy the dot-atom directory to a new .atom directory

### ISSUE THREAD

**Issue by segevfiner** [atom/atom#16531]
### Prerequisites

* [x] Put an X between the brackets on this line if you have done all of the following:
    * Reproduced the problem in Safe Mode: https://flight-manual.atom.io/hacking-atom/sections/debugging/#using-safe-mode
    * Followed all applicable steps in the debugging guide: https://flight-manual.atom.io/hacking-atom/sections/debugging/
    * Checked the FAQs on the message board for common solutions: https://discuss.atom.io/c/faq
    * Checked that your issue isn't already filed: https://github.com/issues?utf8=✓&q=is%3Aissue+user%3Aatom
    * Checked that there is not already an Atom package that provides the described functionality: https://atom.io/packages

### Description

Atom 1.23.2 doesn't copy the [dot-atom](https://github.com/atom/atom/tree/master/dot-atom) directory to a new .atom directory on Ubuntu 16.04 (Atom installed via deb package).

### Steps to Reproduce

1. Move aside your `.atom` directory with Atom closed.
2. Open Atom.
3. Look at the new `.atom` directory.

**Expected behavior:** It should contain the template files from [dot-atom](https://github.com/atom/atom/tree/master/dot-atom).

**Actual behavior:** It doesn't. The config files start out empty.

**Reproduces how often:** 100%

### Versions

```
Ubuntu 16.04.3 x64

Atom    : 1.23.2
Electron: 1.6.15
Chrome  : 56.0.2924.87
Node    : 7.4.0

apm  1.18.12
npm  3.10.10
node 6.9.5 x64
atom 1.23.2
python 2.7.12
git 2.15.1
```

### Additional Information
Atom was installed from `atom-amd64.deb`.

**Comment by tigefa4u** [comment]
@daviwil why xenial not listed here
https://packagecloud.io/AtomEditor/atom

**Comment by rsese** [comment]
Going through some older issues - just noting that I can still reproduce on Ubuntu 18.04 with Atom 1.32.0 but I also noted that the behavior is the same on macOS 10.12.6 with the same version.

**Comment by sadick254** [comment]
Thanks for reporting this!

This has already been fixed in the latest version of atom `v1.51.0`. Please consider upgrading to the latest version.
